docs/release_notes/v1.0.0-rc.3.md

```diff
@@ -0,0 +1,94 @@
+# Spice v1.0-rc.3 (Dec 30, 2024)
+
+Spice v1.0.0-rc.3 is the third release candidate for the first major version of Spice.ai OSS. This release continues the focus on production readiness and includes new Iceberg Catalog APIs, DuckDB improvements, and a new Iceberg Data Connector.
+
+## Highlights in v1.0-rc.2
+
+- **Iceberg Catalog APIs:**
+
+- **Iceberg Data Connector:**
+
+- **DuckDB Data Accelerator Improvements:**
+
+## Breaking changes
+
+- API:`v1/datasets/sample` has been removed as it is not particularly useful, can be replicated via SQL, and via the tools endpoint `POST v1/tools/:name`.
+
+## Dependencies
+
+- [OpenTelemetry](https://opentelemetry.io/): Upgraded from 0.26.0 to 0.27.1
+- [Go](https://go.dev/): Upgraded from 1.22 to 1.23 (CLI)
+
```

Comment:
`datafusion-federation` updated to `ce91d5a6479465ba349f95e3775653e7c27a9185`
`datafusion-table-providers` updated to `19751e31a39c41a454f5b40ef8f1409812769314`
`datafusion` updated to `ffc3ca6bef06a97dbe5c4da1e507ce224511998d`

might need to get updated again if https://github.com/spiceai/spiceai/pull/4064 merges

Reply:
I don't think we need to call out those dependency updates, they are effectively our code.